waku/informational/27/peers.md

```diff
@@ -55,7 +57,7 @@ disconnected gracefully.
 - **`Connected`**: The client is actively connected to this peer.
 
 This list does not preclude clients from tracking more advanced connectivity metadata,
-such as a peer's blacklist status (see [`18/WAKU2-SWAP`](../../standards/application/18/swap.md)).
+such as a peer's blacklist status (see [`18/WAKU2-SWAP`](/waku/deprecated/18/swap.md)).
```

Comment:
I would just remove these two lines. It refers to a deprecated implementation of SWAP